src/appleseed/renderer/modeling/bsdf/orennayarbrdf.cpp

```diff
@@ -148,6 +158,8 @@ namespace
                 sample.m_value.m_diffuse *= values->m_reflectance_multiplier * RcpPi<float>();
             }
             sample.m_value.m_beauty = sample.m_value.m_diffuse;
+            
+            sample.m_max_roughness = values->m_roughness;
```

Comment:
We can probaby set m_max_roughness to 1 for this BRDF and use the roughness the user specified.
Doing the same as in the Lambertian BRDF.